docs/data/material/components/transitions/SimpleCollapse.tsx

```diff
@@ -4,21 +4,20 @@ import Switch from '@mui/material/Switch';
 import Paper from '@mui/material/Paper';
 import Collapse from '@mui/material/Collapse';
 import FormControlLabel from '@mui/material/FormControlLabel';
-import { Theme } from '@mui/material/styles';
 
 const icon = (
-  <Paper sx={{ m: 1 }} elevation={4}>
-    <Box component="svg" sx={{ width: 100, height: 100 }}>
+  <Paper sx={{ m: 1, width: 100, height: 100 }} elevation={4}>
+    <svg>
```

Comment:
Move the style closer to where they make sense, also kill one React component along the way Box -> svg.